### ISSUE TITLE
Uncaught ReferenceError: minJavaRuntime is not defined

### ISSUE THREAD

**Issue by lindad2000** [cqframework/atom_cql_support#32]
This message appeared when I opened Atom.

1. ...
2. ...

**Atom**: 1.55.0 x64
**Electron**: 6.1.12
**OS**: Microsoft Windows 10 Enterprise
**Thrown From**: [language-cql](https://github.com/cqframework/atom_cql_support) package 2.9.0

### Stack Trace

Uncaught ReferenceError: minJavaRuntime is not defined

```
At C:\Users\linda.duffy\.atom\packages\language-cql\language-cql\pkg\language-cql-common\lib\java-helpers.js:65

ReferenceError: minJavaRuntime is not defined
    at showJavaRequirements (/packages/language-cql/language-cql/pkg/language-cql-common/lib/java-helpers.js:65:136)
    at /packages/language-cql/language-cql/pkg/language-cql-common/lib/java-helpers.js:11:13)
    at ChildProcess.emit (events.js:200:13)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:270:12)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:9)
```

### Commands

```
```

### Non-Core Packages

```
atom-ide-ui 0.13.0 
language-cql 2.9.0 
```

**Comment by JPercival** [comment]
Whoops. Auto closed. This is fixed in the next release. In the meantime, please make sure you have Java >=8 installed and that that your JAVA_HOME environment variables are set correctly.

Installing Java:
https://adoptopenjdk.net/?variant=openjdk11&jvmVariant=openj9
Setting Java Path:
https://www.java.com/en/download/help/path.xml

**Comment by JPercival** [comment]
Fixed with release 2.9.1
